### ISSUE TITLE
Configure Ranger Species (plants)

### ISSUE THREAD

**Issue by pgoellnitz** [globe-swiss/phaenonet-client#172]
Configure the new plant species for the rangers.

- [x] add species configuration
  - [x] add "wild garlic" species
- [x] add translations
- [x] optimize and add icons
- [x] add/handle limits for new species
- [ ] validate configuration by WSL

Open:
- [x] define the new species to configure @daria-sl 
- [x] translate all new text @daria-sl 
- [x] icon pack of the map pins @daria-sl 
- [x] verify and update 'Bärlauch' species

Current configuration format for species:
![image](https://user-images.githubusercontent.com/17765935/131116846-541d1111-7277-41da-9bc3-ceec2e395732.png)

**Comment by daria-sl** [comment]
Here go the **plant** species to introduce with their abbreviations - Dani is working on the icons.

Spezies_de | Spezies_fr | Spezies_it | Abkürzung_de | Abkürzung_fr | Abkürzung_it
-- | -- | -- | -- | -- | --
Gemeines Pfaffenhütchen | Fusain d'Europe | Berretto da prete | Gp | Fe | Be
Vogelkirsche | Merisier | Ciliegio | Vk | Mr | Ci
Trauben-Eiche | Chêne sessile | Quercia rovere | Te | Cs | Qr
Stiel-Eiche | Chêne pédonculé | Quercia comune | Se | Cd | Qc
Flaumeiche | Chêne pubescent | Quercia pubescente | Fe | Cb | Qp
Heidelbeere | Myrtille | Mirtillo nero | He | My | Mi
Eingriffeliger Weissdorn | Aubépine monogyne | Biancospino comune | Ew | Am | Bc
Bärlauch | Ail des ours | Aglio orsino | Bl | Ao | Ao
Hartriegel | Cornouiller sanguin | Corniolo sanguinello | Hr | Co | Cs
Waldrebe | Clématite blanche | Clematide vitalba | Wr | Cb | Cv

The insects will follow in a second step.

**Comment by pgoellnitz** [comment]
@daria-sl: I would assume that we don't set any limits on these new species for the rangers, right?

**Comment by daria-sl** [comment]
good question. I will ask the WSL on our next meeting. But so far I guess there are no limits.

**Comment by pgoellnitz** [comment]
while your on it, we also have no comments defined for the phenology phases - if that's right ;)

**Comment by pgoellnitz** [comment]
@daria-sl, as I see that the insect part is still not ready im splitting this part of to #185. Let's put all insect info there.

**Comment by daria-sl** [comment]
@pgoellnitz the Bärlauch is now completed.
For the limits: no limits to be introduced for PhenoRanger species
For the comments: no pre-defined comments to be introduced for PhenoRanger species. However, the WSL would wish to have a comment field where Rangers could enter any desired text. As this is not part of our offer for them, it is not a priority - still, I'd be happy to know from your side if such an implementation would be time-consuming or rather not, so we could give an according feedback to WSL about their wish.

**Comment by pgoellnitz** [comment]
@daria-sl: create a separate item with PhenoRanger label. Let's discuss this once the initial scope is done.
